### PR TITLE
Improvement/legacy support

### PR DESCRIPTION
**Pull request by kazoompa**

### CODE REVIEW

File: obiba_mica/import_zip.py [hunk 1]
```diff
@@ -3,65 +3,145 @@
...
+
+    @staticmethod
+    def needsLegacySupport(client, verbose=False):
+        restService = RestService(client, verbose)
+        response = restService.send_request("/auth/session/_current", restService.make_request("GET"))
+        # Only the Mica server supporting Java 21 has version information in the response
+        versionInfo =  response.version_info
+
+        if versionInfo is not None:
+            return int(versionInfo["major"]) + int(versionInfo["minor"]) < 10
```
Comment (ymarcon) [inline obiba_mica/import_zip.py]: What does that sum means?

Comment (ymarcon) [reply obiba_mica/import_zip.py]: Now that it passes the checks, can you explain this one?

Comment (kazoompa) [reply obiba_mica/import_zip.py]: I have changed  the code to be more readable, the old code meant to check for `vesions < 5.5.x`.

Comment (ymarcon) [reply obiba_mica/import_zip.py]: But 6 + 0 is lower than 10

Comment (kazoompa) [reply obiba_mica/import_zip.py]: This code is replaced by:

```
return int(versionInfo["major"]) <= 5 and int(versionInfo["minor"]) < 5
```

**Pull request merged**